apps/src/music/views/ExemplarPlayerView.tsx

```diff
@@ -46,6 +46,14 @@ const ExemplarPlayerView: React.FunctionComponent<ExemplarPlayerViewProps> = ({
     dispatch(setIsPlaying(false));
     playerRef.current?.stopSong();
 
+    const currentLibrary = MusicLibrary.getInstance();
+    if (currentLibrary) {
+      playerRef.current?.updateConfiguration(
```

Comment:
As a follow-up we may want to consider just passing in the music player reference as a prop into this component; that way, we don't have to separately set the BPM and key here. Since we don't want the exemplar and main player to ever be playing at the same time, sharing the same instance may also help enforce that behavior.